tests/unit/time_series_test.coffee

```diff
@@ -71,3 +72,168 @@ test 'Margins are the right size when showLegend is no', (assert) ->
   assert.equal @subject().get('marginRight'), 0, 'no right margin'
   assert.equal @subject().get('marginBottom'), 0,
     'no bottom margin if showLegend is no'
+
+test 'The number of labels should be equal to the number of day intervals',
+(assert) ->
+  timeSeriesContentLine =
```

Comment:
you define the `timeSeriesContentLine` many times even though it's only slightly different each time,  could you define it up top and change it however you need as you go?